nengo_viz/static/viz_slider.js

```diff
@@ -152,8 +164,8 @@ VIZ.Slider.prototype.constructor = VIZ.Slider;
 
 VIZ.Slider.prototype.set_value = function(slider_index, value, immediate) {
     console.assert(typeof slider_index == 'number');
-    console.assert(typeof value == 'number');
-    console.assert(typeof immediate == 'boolean');
+    console.assert(typeof value == 'number', value);
+    //console.assert(typeof immediate == 'boolean');
```

Comment:
Argh, nope, I need to clear that up -- I added a third option, so it's not a boolean any more....  I'll add something to make that clearer.